ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -63,15 +63,12 @@
         {{#if (and (eq this.mode "performance") (not (has-feature "Performance Replication")))}}
           <p>
             Performance Replication is a feature of Vault Enterprise Premium.
-          </p>
-          <p class="has-text-centered">
             <ExternalLink
               @href="https://hashicorp.com/products/vault/trial?source=vaultui_Performance%20Replication"
-              class="button is-ghost has-icon-right"
-              data-test-upgrade-link
+              class="link"
```

Comment:
If we're editing these perhaps we should update the copy to move away from the generic `Learn more` text